docker-compose.yml

```diff
@@ -1,9 +1,14 @@
-version: '3'
+version: "3"
 services:
+  client:
+    image: openmind/pis-client:latest
+    build: ./client/
+    ports:
+      - 80:80
   server:
-    image: openmined/pis:latest
+    image: openmined/pis-server:latest
     build: ./server/
-    volumes: ['./server:/server']
+    volumes: ["./server:/server"]
```

Comment:
Yikes, this is from my IDE. Y'all have a preference or codestyle enforcement mechanism?

Reply:
Not yet! This is fine, though :) thanks!